src/sentry/api/event_search.py

```diff
@@ -1561,25 +1576,34 @@ def validate_argument_count(self, field, arguments):
         ),
         Function(
             "p50",
-            aggregate=[u"quantile(0.5)", "transaction.duration", None],
+            optional_args=[with_default("transaction.duration")(NumericColumnNoLookup("column"))],
```

Comment:
I like the with_default() function but think the currying is a bit elaborate. Wouldn't `with_default('transaction.duration', NumericColumnNoLookup('column'))` work just as well?